src/pages/memberhsip/components/youthProfileForm/YouthProfileForm.tsx

```diff
@@ -207,9 +221,53 @@ function RegistrationForm(props: Props) {
                 as={TextInput}
                 id="otherLanguages"
                 name="otherLanguages"
-                //type="hidden"
+                type={props.values.language.includes('Other') ? '' : 'hidden'}
               />
             </div>
+            <span
```

Comment:
Missed this at first review, you should use something else than a `span` here, e.g. a `div`